app/src/forms/form/exportService.js

```diff
@@ -133,7 +133,29 @@ const service = {
     };
   },
 
+  _readLatestFormSchema: async (formId) => {
+    return FormVersion.query()
+      .select('schema')
+      .withGraphFetched('submissions')
```

Comment:
Does this need a join on submissions? Are we just getting the form version.
(actually important to note withGraphFetched isn't a join it's a second query that does a WHERE IN on the parent results, so big time increase if it's needed)

Reply:
ok. good to know.. i thought this might be unnecessary. The idea was to only find the latest form version that has submissions. but you're right.. this could be a big overhead with a lot of submissions. Guess we should remove this clause and just get latest form version, regardless of whether it has submissions